app/sync/go-to.js

```diff
@@ -59,6 +61,24 @@ exports.create = function (api) {
         page.id = page.id || locationId
         tabs.add(page, !openBackground, split)
 
+        // Save Tabs: If enabled then add it to default tabs.
+        const saveTabs = api.settings.obs.get('patchbay.saveTabs', false)
+        if (saveTabs()) {
+          const openTabs = api.settings.obs.get('patchbay.openTabs', [])
+          var _tabs = openTabs()
```

Comment:
If using const, why not use `let` instead of `var` to be a tiny bit more strict regarding variable access?

Reply:
Thx... will do.
I "think" I can use api.settings.sync.get rather than obs.get too